src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -188,16 +167,6 @@ public DynamoAnalyticsClient(DynamoModel dynamoModel)
             product = new ProductInfo() { Id = "DYN", Name = hostName, VersionString = appversion, AppVersion = appversion, BuildId = buildId, ReleaseId = releaseId };
         }
 
-        private void RegisterGATracker(Service service)
```

Comment:
seems we want this.

Reply:
Tacker is the center piece shared by GA and instrumentation